extends Node2D
#CreaciÃ³n de una funciÃ³n â€“ Mensaje repetido
func gohl() -> void:
	print ("!HeyÂ¡!EnanoÂ¡!Este es MI puenteÂ¡")

#CreaciÃ³n de una funciÃ³n con retorno â€“ Hora actual
var dateTimeStamp_actual = Time.get_datetime_dict_from_system()
var horas = str(dateTimeStamp_actual.hour)
var minutos = str(dateTimeStamp_actual.minute)
var segundos = str(dateTimeStamp_actual.second)
var hora_actual = horas + ":" + minutos + ":" + segundos
func hora() -> int:
	return hora_actual

#CreaciÃ³n de una funciÃ³n parametrizada â€“ Mensaje segÃºn clase
func jugador(clase_jugador) -> void:
	if clase_jugador.to_lower() == "paladin":
		print("Oh ya veo, con que un PaladÃ­n eh?, dime, que te motivÃ³ a escoger un camino tan... aburrido?")
	elif clase_jugador.to_lower() == "berserk":
		print ("Hacha, guerra, hacha, guerra... asi hablan ustedes, no?")
	elif clase_jugador.to_lower() == "mago":
		print ("No creo que tengamos algo que te guste, aqui solo tenemos espadas y hachas gigantes. â„n Oh! espera, creo que encontrÃ© esta rama en mi jardin")
	elif clase_jugador.to_lower() == "druida":
		print ("DIOS que hedor. Espera... no me digas que ese eres tu??")
	else:
		print ("Con que no te has decidido que ser aun, eh? No te preocupes, aqui no hay ninguna luna con cara malevola que te apurarÃ¡")


func _ready() -> void:
	gohl()
	print("*Pateas al troll y sale volando. Ahora es una estrella mas en el cielo*")
	print("+Subes de nivel+")
	jugador("berserk")
#Arreglo Simple â€“ Dialogo de enemigos
	var enemigos = ["Â¡Ostia tio!!Carla prepara la olla hoy se come!","Soy herbivoro pero tengo hambre, perdon ğŸ¤·â€â™‚ï¸"]
	enemigos.append("!Saludame a la abuela cocoÂ¡")
	enemigos.append("Ciclo alimenticio nene ğŸ’…")
#EXTRA
	print ("*Recolectas bayas en el bosque para el camino. Escuchas algo detenerse a pasos de ti, para cuando te volteas el ogro que te miraba con ojos brillantes y grita*")
	print(enemigos[0])
	print("*No miras al frente por unos segundos mientras corrias y terminas en picada por un acantilado*")
	print("*Afortunadamente caes sobre unos arboles, pero se te rompio el dedo meÃ±ique del pie, Â¿Como?, no lo se*")
#Arreglo Simple â€“ Inventario Simple
	var inventario=["bayas","flechas","pocion de vida","bombas","stickers"]
	print("Usaste una pociÃ³n de vida")
	inventario.remove_at(2)
